zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -83,13 +83,18 @@ zipkin:
     # - .*example2.com
     # Default is "match all websites"
     instrumented: .*
+  server:
```

Comment:
agreed. we should re-use zipkin.query as I don't expect us to apply this to other components such as the /health endpoint